.travis.yml

```diff
@@ -122,6 +122,20 @@ jobs:
         - travis_wait make test-cmd-app
         - travis_wait make test-cmd-push
         - travis_wait make test-cmd-project
+        - odo logout
+
+    - <<: *base-test
+      stage: test
+      name: "devfile catalog, create, push, delete and docker devfile url command integration tests"
+      script:
+        - ./scripts/oc-cluster.sh
```

Comment:
Shouldn’t we be removing this? That is oc cluster setup for docker and kube?

Reply:
I did not get the context of what you expecting/asking, Are you looking a separate job for this ? can you please explain bit more ?

Actually my thought was, without creating a separate job how i can optimize/rearrange the test file for docker push target test with the actual kube config file. For example ```test-cmd-docker-devfile-url``` needs kube config file for certain scenario. So what i did i just move the test location to a place where it needs a fresh kube config. 

However i kept the rest of test file such as ```test-cmd-docker-devfile-push```, ```test-cmd-docker-devfile-catalog``` and ```test-cmd-docker-devfile-delete``` in a separate job without a Kube environment i mean no ```oc cluster up```.

Optimization it brings
- Use the kube file without creating a separate job.
- Remove oc cluster up for test that does not need Kube environment. (Time decreased by one oc cluster up)

Reply:
@girishramnani As per our slack discussion i will update the comment message. Thanks for the feedback

Reply:
Updated
ping @girishramnani